decidim-admin/app/packs/stylesheets/decidim/admin/modules/_forms.scss

```diff
@@ -116,6 +116,14 @@ label,
   margin-bottom: $input-margin;
 }
 
+.hashtags__container{
```

Comment:
I think we should not wrap this fix only for the hashtags container because then the same errors still exists for inputs that:
1. Have character counter
2. Have dynamic errors
3. DO NOT have hashtags

The approach also has another problem is that when the error message is long or the screen width is very small, it would still overlay on top of the character counter. You can try this locally by toggling the "device toolbar" from the browser's developer tools and enable some narrow screen mobile device from the list. After this, using the inspect element, edit the error message to contain a lot more characters and you should see something like this as the end result:

![Long error messages on narrow screens](https://user-images.githubusercontent.com/864340/184349958-fd5eb5ae-b0dc-4cf4-86cb-71dbaf3af93d.png)

I think this same "fix" for the same issue we are using in the participant side should also work for the admin panel:
https://github.com/decidim/decidim/blob/ab994a55655a1b3e7e23b830fdf4e4492ad95a49/decidim-core/app/packs/stylesheets/decidim/modules/_forms.scss#L53-L55

But since the admin panel styling is a bit different, you'll need to multiply it by `1.5` instead of `.5` as in the participant side.

Another thing you need to do is to make the character counter to display as a block, the same way as we have for the participant side:
https://github.com/decidim/decidim/blob/ab994a55655a1b3e7e23b830fdf4e4492ad95a49/decidim-core/app/packs/stylesheets/decidim/modules/_forms.scss#L43

And finally, I would also apply the same top margin "fix" as in the participant side for the counter to appear a bit closer to the input element:
https://github.com/decidim/decidim/blob/ab994a55655a1b3e7e23b830fdf4e4492ad95a49/decidim-core/app/packs/stylesheets/decidim/modules/_forms.scss#L46

Could you do that instead?

The end result should look something like this:
![Expected result](https://user-images.githubusercontent.com/864340/184351211-c8d1561b-a6c4-443d-8d91-c3478ed98ff5.png)

I just added the long error there using inspect element for demonstration purposes. I know it's shorter in this particular case but this fix should apply everywhere in the admin panel.